### PR TITLE
[mono] Fix https://github.com/dotnet/runtime/issues/72519.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: src/tests/JIT/Regression/JitBlue/Runtime_71592/test71592.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public class Test
         var t1 = new Test () { foo = 1 };
         var t2 = new Test () { foo = 2 };
         if (t1 == t2)
-            return 0;
+            return 100;
         return 100;
```
Comment (kasperk81) [inline src/tests/JIT/Regression/JitBlue/Runtime_71592/test71592.cs:31]:
```suggestion
        return 101;
```
when it should fail?

Comment (vargaz) [reply src/tests/JIT/Regression/JitBlue/Runtime_71592/test71592.cs:31]: This was supposed to be a test for an infinite loop in an optimization pass, so
the == call is only there to make sure the problematic code is called.

**Review by lambdageek — APPROVED** [review]

**Comment by lambdageek** [comment]
@vargaz I think we should make this a Pri0 test  (change CLRTestPriority here):

https://github.com/dotnet/runtime/blob/8ed99c81c4d79e9da166281ffda35d0adde30401/src/tests/JIT/Regression/JitBlue/Runtime_71592/test71592.csproj#L4 

I'm not sure we run Pri1 runtime tests on any pipeline for mono.  Do you know @steveisok - is there some outerloop that runs Pri1 runtime tests with mono? I couldn't find one.

**Comment by steveisok** [comment]
> I'm not sure we run Pri1 runtime tests on any pipeline for mono.  Do you know @steveisok - is there some outerloop that runs Pri1 runtime tests with mono? I couldn't find one.

Not that I know of.

**Review by jakobbotsch — APPROVED** [review]

**Pull request merged**